src/_Problems_/get-mazePath/index.js

```diff
@@ -0,0 +1,42 @@
+//======================================Problem Statement=============================================
+// --->> Print all possible path to reach the end of the GRID/MAZE (N x N) from starting  point to ending point
+// --->> One horizontal move will be represented by H and one vertical move will be represented by V
+// --->> Complexity =  Complexity will be exponential as there are many overlapping solutions
```

Comment:
Complexity for this should be :
2^n where n = No of cells between start point to end point.
Example: 
- For (0,0) -> (2,2) , n= 9
- For (0,0) -> (0,2) , n= 3